template.md

```diff
@@ -34,8 +34,9 @@ Deadline for advancement eligibility: [**CALCULATE TIME HERE**](https://www.time
     1. Such proposals *should* include supporting materials when possible.
 1. Proposals looking to advance to stage 1 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
     1. Such proposals *must* link to a proposal repository and they *should* link to supporting materials when possible.
-1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
-    1. Such proposals *must* link to supporting materials prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline. If these materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
+1. Proposals looking to advance to stages 2, 3, or 4, as well as other normative changes to the standard or proposals in stage 3 or later looking to achieve consensus, must be added (and noted as such) *along with links to the necessary materials* prior to the deadline, or else delegates may withhold consensus solely on the basis of missing the deadline.
+    1. If the supporting materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
```

Comment:
> may withhold consensus
> may be disqualified from advancement

is there a substantive difference between these two?  or should this language be normalized?

Reply:
@ptomato

Reply:
Sorry, missed the earlier ping. Looking back at the earlier revisions, I changed the above "object to advancement" to "withhold consensus" based on [this comment](https://github.com/tc39/agendas/pull/1073#discussion_r734928622).

The "may be disqualified from advancement, based on the committee's judgement" language is pre-existing, but on further reflection it's not really clear how that takes place — does the committee have to reach consensus on judging that the materials changed substantially? I think it makes sense to change _something_ here, but changing it to something along the lines of "one delegate may withhold consensus if the materials change substantially" seems like it'd incentivize haggling over whether a change is "substantial" or not...

Reply:
imo it's subjective, so it's up to each delegate. If an individual considers it substantial, it is.

Reply:
- the `may be disqualified` language suggests there is some objective criteria that would disqualify something automatically.  
- agree with Jordan's comment
- I think we should just normalize this language, as it amounts to the same thing, AFAICT

Reply:
resolved via #1362 if we agree

Reply:
Sure, works for me. I was concerned about opening up an avenue for abuse of the process, but that avenue was kind of already there.